src/Plugins/DBFTPlugin/Consensus/ConsensusContext.cs

```diff
@@ -168,7 +170,7 @@ public Block EnsureHeader()
 
         public bool Load()
         {
-            byte[] data = store.TryGet(ConsensusStateKey);
+            byte[] data = store?.TryGet(ConsensusStateKey);
             if (data is null || data.Length == 0) return false;
```

Comment:
if this return false it could affect `ConsensusService`. (I think that is not possible)

Reply:
Can we do something like change all Tryxxxx thing,,, make it return false or true

Reply:
This store is only for recovery.

https://github.com/neo-project/neo/blob/7e639a1318251d5da5eb237696d205e20719dea4/src/Plugins/DBFTPlugin/Consensus/ConsensusService.cs#L148-L162